css/style.css

```diff
@@ -0,0 +1,40 @@
+body{
+    background: #7373BC;
+    color: white;
+    text-align: left;
+}
+
+#main-title{
+    text-align: center;
+}
+
+#description {
+    font-weight: bold;
+}
+
+#survey-form{
+    display: flex;
+    flex-direction: column;
+    background: #2B2B49;
+    margin-left: 25%;
+    margin-right: 25%;
+    padding: 2%;
+}
+
+@media (max-width: 768px){
+    #survey-form{
+        margin-left: 5%;
+        margin-right: 5%;
+    }
+}
+
+#submit {
+    width: 10em;
+    height: 2em;
+    align-self: center;
+    font-size: 1.1em;
+    margin-top: 5%;
+}
+ #submit:hover {
+    transform: scale(1.2);
+ }
```

Comment:
Try to get installed the linter files that allow you to know if your style file follows the best practices.

Reply:
- [ ] Please make the comments polite and easy to understand. Using a polite tone makes communication smoother while easy-to-understand comments would help the author understand what changes need to be made. 😄📗 

 #### Example of Polite comments 
 
> **_Kindly_** use Stylelint linters for this project. 
> I think it would be better to use Stylelint linters for this project. 

#### Example of a clear and easy-to-understand comment

 > **_Kindly_** use Stylelint linters for this project. Stylelint reports bad codes in your CSS files. It helps to enforce consistent code and prevents you from making errors in your stylesheets. 👍 😎 
 
 #### Hint: You don't have to create another inline comment. You could just edit the inline comments that you have above. 💻